src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -19,20 +19,17 @@ object StringUtils {
     }
 
     fun String.removeColor(): String {
-//        return replace("(?i)\\u00A7.", "")
-
         val builder = StringBuilder()
```

Comment:
Also: we can initialize this StringBuilder with a capacity
```suggestion
        val builder = StringBuilder(this.length)
```